### DISCUSSION TITLE
Adding New Building Props from 3DS Max

### DISCUSSION THREAD

**Discussion by l1th1um** [carla-simulator/carla#4619] [Q&A]
Hi,

I'm trying to add new building props created using 3Ds Max then import it as fbx (https://carla.readthedocs.io/en/0.9.12/tuto_A_add_props/). Using command **make import**, it returns FileNotFoundError: [Errno 2] No such file or directory: '\carla\Import\roadpainter_decals.json'. I couldn't find this file in my Installation.  

This is my folder structure
```

D:\carla
└── Import
	│
	├── Jambi
		   ├── Jambi.json
		   └── Props
	    	            ├── Building1
	        	    ├── model_2021.jpg
	          	    └── Building1.fbx


```
This is the content of **Jambi.json**

```
{
  "maps": [
  ],
  "props": [
    {
      "name": "Building1",
      "size": "medium",
      "source": "./Props/Building1/Building1.fbx",
      "tag": "Building"
    }    
  ]
}
```

**Comment by yujiayong123** [comment]
now ok?
